admin/client/src/legacy/LeftAndMain.js

```diff
@@ -370,15 +370,15 @@ $.entwine('ss', function($) {
      * @param object data - Any additional data passed through to `window.history.state`.
      * @param boolean forceReload - Forces the replacement of the current history state, even if the URL is the same, i.e. allows reloading.
      */
-    loadPanel: function (url, title = '', data = {}, forceReload, forceReferer = window.history.state.path) {
+    loadPanel: function (url, title = '', data = {}, forceReload, historyState = window.history.state) {
```

Comment:
This does make loadPanel a lot harder to use; Before you could pass in a string to the 4th parameter, but now it has to be a state. It seems to be a case of API that serves the api itself, rather than the human using it.